src/xmrEngine.js

```diff
@@ -611,6 +611,31 @@ export class MoneroEngine {
     return false
   }
 
+  async createMyMoneroTransaction(
+    options: CreateTransactionOptions
+  ): Promise<CreatedTransaction> {
+    let result: CreatedTransaction
+    try {
+      result = await this.myMoneroApi.createTransaction(
```

Comment:
You can just do `return await` and skip the whole `result` thing, assuming lint is up to date.